src/components/TitleBar.ts

```diff
@@ -4,11 +4,11 @@ import { Link } from 'src/components/common';
 import { icon } from 'src/components/icons';
 
 interface ITitleBar {
```

Comment:
This is border-line in scope, so might as well rename this interface to match conventions.

```suggestion
interface TitleBarProps {
```